tests/safety/common.py

```diff
@@ -190,6 +190,7 @@ class TorqueSteeringSafetyTestBase(PandaSafetyTestBase, abc.ABC):
 
   @classmethod
   def setUpClass(cls):
+    super().setUpClass()
```

Comment:
don't think this does anything. the endswith check is in `PandaSafetyTest`, not the base class. forgot why I didn't just put endswith base in the `PandaSafetyTestBase` class, can you see if there's any issues? can be separate from this subaru-specific PR